docs/about/citation.rst

```diff
@@ -3,36 +3,43 @@
 Acknowledging and Citing
 ========================
 
-If you use PlasmaPy for a project that results in a research publication,
-we ask that you cite the `Zenodo <https://zenodo.org>`__ record for the
-specific version of PlasmaPy used in your project.  Citing a software
-package promotes scientific reproducibility, gives credit to its developers,
-and highlights the importance of software as a vital research product.
+.. |version_to_cite| replace:: 0.8.0
+.. |doi_hyperlink| replace:: https://doi.org/10.5281/zenodo.6774350
```

Comment:
When I go to this link I get a "DOI Not Found"?  Has this doi only been reserved and not yet "created"?

Reply:
Correct: the link to the DOI does not exist yet, so this link won't work until after we've created the Zenodo record.  This does make it annoying to test, though...

Reply:
Ok, just wanted to double check before approving.